pyalgotrading/algobulls/connection.py

```diff
@@ -972,24 +1145,32 @@ def get_papertrading_report_statistics(self, strategy_code, initial_funds=1e9, m
         else:
             print('Generating Statistics for already fetched P&L data...')
 
+        if initial_funds is None:
+            initial_funds = self.saved_parameters.get("initial_funds_virtual") or 1e9
+
         order_report = self.get_report_statistics(strategy_code, initial_funds, report, html_dump, self.papertrade_pnl_data)
 
         return order_report
 
-    def get_papertrading_report_order_history(self, strategy_code):
+    def get_papertrading_report_order_history(self, strategy_code, country=None, render_as_dataframe=False):
         """
         Fetch Paper Trading order history
 
         Args:
-            strategy_code: Strategy code
+            strategy_code: strategy code
+            country: country of the segment
+            render_as_dataframe: return order history as dataframe or pretty string
 
         Returns:
             Report details
         """
 
         assert isinstance(strategy_code, str), f'Argument "strategy_code" should be a string'
 
-        return self.get_report(strategy_code=strategy_code, trading_type=TradingType.PAPERTRADING, report_type=TradingReportType.ORDER_HISTORY)
+        if country is None:
```

Comment:
can we move this logic inside get_report? will it break anything? if no, let's do it at other places with similar logic